internal/sleuth/incident_impact_source_resource.go

```diff
@@ -312,6 +322,36 @@ Options: ALL, P1, P2, P3, P4, P5. Defaults to ALL`,
 					},
 				},
 			},
+			"rootly_input": schema.SingleNestedAttribute{
+				Optional:            true,
+				MarkdownDescription: "Rootly input",
+				Attributes: map[string]schema.Attribute{
+					"remote_severity": schema.StringAttribute{
+						Optional:            true,
+						MarkdownDescription: "Rootly’s severity values are configurable, but ultimately they always map to 4 levels: ALL, CRITICAL, HIGH, MEDIUM and LOW. Check out your current [severities configuration in Rootly](https://rootly.com/account/severities).",
```

Comment:
These markdown descriptions are used in the `docs/resources/incident_impact_source.md` via `make docs` and taken out of Sleuth DORA's UI:

<img width="467" alt="rootly create impact source screen" src="https://github.com/user-attachments/assets/c7b8e69d-fd8f-426d-8b33-81f356274543">